Microsoft.Toolkit.UWP.Input.GazeInteraction/Microsoft.Toolkit.Uwp.Input.GazeInteraction.vcxproj.filters

```diff
@@ -0,0 +1,121 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should the .filters file be checked in or should it be in a `.gitignore`?

Reply:
They shouldn't affect the build, so they shouldn't be included in the source control. Ignore it.